campari/tests/test_campari.py

```diff
@@ -105,35 +105,36 @@ def test_findAllExposures(roman_path):
 
 def test_simulate_images(roman_path):
     lam = 1293  # nm
-    band = "F184"
+    band = "Y106"
```

Comment:
I probably would put the definitions of `ra`, `dec`, `sca`, and `pointing` here just to make these all easy to find at the beginning of the test function.